README.md

```diff
@@ -47,11 +47,11 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo]()
+- [Live Demo](https://richardsmarioinfo.netlify.app/)
```

Comment:
- [ ] `Required`
- [ ] Amazing Work so far 👏 
- [ ] However your Netlify deployment is not working as expected, please i suggest you fix it. Just as the Previous reviewer suggested 🥂 
- [ ] please see screenshot below
- [ ] 
![image](https://github.com/richie1988/SuperMario/assets/66230306/880b9a3a-e27e-4fa0-b199-d9c88b382906)